armadillo/src/main/java/org/molgenis/armadillo/ArmadilloUtils.java

```diff
@@ -22,10 +27,13 @@ public static String serializeExpression(String cmd) {
     byte[] rawResult = new byte[0];
     if (result.isRaw()) {
       try {
+        LOGGER.trace(format("RAW result : [ %s ]", new String(result.asBytes(), UTF_8)));
         rawResult = result.asBytes();
       } catch (REXPMismatchException e) {
         throw new IllegalStateException(e);
       }
+    } else {
```

Comment:
Let's just get really upset and throw exception if it's not a raw result. Fail fast, fail hard.